src/scripts/main.js

```diff
@@ -1 +1,15 @@
-console.log("hey");
+   
+    	$(function(){
+    		// On page load / resize, force now & next header boxes to be equal (sqaure) size.
+			var $el = $('.now-next-live');
+			var $el2 = $('.now-next-now');
+			var $el3 = $('.now-next-next');
+			var $window = $(window).on('resize', function(){
+			   var width = $el.outerWidth();
+			   $el.height(width);
+			   $el2.height(width);
+			   $el3.height(width);
+			}).trigger('resize'); //on page load
+
+		});
```

Comment:
As previously discussed (just doing it as part of the fancy github review), are we sure this needs to be javascript? I feel like it doesn't need to be...